Bonsai.Harp.CF/SyringePumpCommand.cs

```diff
@@ -55,20 +53,19 @@ string Description
             {
                 switch(Type)
                 {
-                    case SyringePumpCommandType.EnableMotorDriver: return "Enables the motor on the Syringe Pump";
-                    case SyringePumpCommandType.DisableMotorDriver: return "Disables the motor on the Syringe Pump";
-                    case SyringePumpCommandType.StartProtocol: return "Start the configured protocol on the Syringe Pump";
-                    case SyringePumpCommandType.StopProtocol: return "Stop the running protocol on the Syringe Pump";
-                    case SyringePumpCommandType.SetDigitalOutputs: return "Set the state of digital output 0 using the input boolean value.";
-                    case SyringePumpCommandType.ClearDigitalOutputs: return "Set the state of digital output 1 using the input boolean value.";
-                    case SyringePumpCommandType.MotorMicrostep: return "Set the motor microstep value.";
-                    case SyringePumpCommandType.ProtocolNumberOfSteps: return "Set the number of steps to run in the protocol [1;65535]";
-                    case SyringePumpCommandType.ProtocolStepsPeriod: return "Set the period in ms between each step on the protocol [1;65535]";
-                    case SyringePumpCommandType.ProtocolFlowRate: return "Set the flow rate of the protocol [0.5;2000.0]";
-                    case SyringePumpCommandType.ProtocolVolume: return "Set the volume in uL of the protocol [0.5;2000.0]";
-                    case SyringePumpCommandType.ProtocolType: return "Set the type of the protocol. False for step-based and True to volume-based protocol.";
-                    case SyringePumpCommandType.CalibrationValue1: return "Set the calibration value 1 for protocol use.";
-                    case SyringePumpCommandType.CalibrationValue2: return "Set the calibration value 2 for protocol use.";
+                    case SyringePumpCommandType.SetMotorDriver: return "Enables/disables the motor on the Syringe Pump.\n\n[Input]\nExpects a boolean.\n\n'true' enables the motor\n'false' disables the motor.";
```

Comment:
Hopefully we will find a good way to specify allowable inputs, but this is okay for now.